.scripts/Run-Checks.ps1

```diff
@@ -80,6 +80,7 @@ function Run-TopLevelNamespaceCheck {
         "ClientPipelineExtensions.cs",
         "ClientUriBuilder.cs",
         "ErrorResult.cs",
+        "IEnumerableExtensions.cs",
```

Comment:
Could you also update the version of System.ClientModel that is referenced in the .scripts/Export-API.ps1 script here:
🔗 https://github.com/joseharriaga/openai-in-typespec/blob/main/.scripts/Export-API.ps1#L23

"1.1.0-beta.5" --> "1.1.0"